### PR TITLE
Bump plan evaluator version

### PR DESCRIPTION
**Pull request by vincent-karuri**

### CODE REVIEW

File: src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java [hunk 1]
```diff
@@ -427,6 +427,11 @@ public Task updateTask(Task task) {
 		return convertToFHIRTasks(convert(taskMetadataMapper.selectMany(example, 0, DEFAULT_FETCH_SIZE)));
 	}
 
+	@Override
```
Comment (Rkareko) [inline src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java:430]: Could we add some javadocs here to explain why null is returned

Comment (vincent-karuri) [reply src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java:430]: I think we should make it similar to https://github.com/opensrp/opensrp-server-core/blob/81b0984ab1468f1f271167aa9ff83a80f15fc68d/src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java#L424.

**Comment by Rkareko** [comment]
@vincent-karuri the plan evaluator version has already been incremented in this PR https://github.com/opensrp/opensrp-server-core/pull/450/files

**Comment by vincent-karuri** [comment]
cool

**Pull request closed without merging**